src/InstrumentationEngine/ProfilerManagerWrapper.h

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "ProfilerManager.h"
+
+namespace MicrosoftInstrumentationEngine
+{
+
+    class CProfilerManagerWrapper :
+        public IProfilerManager,
+        public IProfilerManager2,
+        public IProfilerManager3,
+        public IProfilerManager4,
+        public IProfilerManager5,
```

Comment:
>IProfilerManager5 [](start = 15, length = 17)

so if we add IProfilerManager6, this class won't be able to QI for it by default. Might be worthwhile to add comments into the idl to make sure this is updated as well.

Reply:
I've added the comment in CProfilerManager since we only want to update CProfilerManagerWrapper if the Manager is updated.

---
In reply to: [331700175](https://github.com/microsoft/CLRInstrumentationEngine/pull/146#discussion_r331700175) [](ancestors = 331700175)